src/PlayPauseStop/ScoreStop.qml

```diff
@@ -20,14 +20,14 @@ Button {
     // Change the button color when it is pressed
     onPressed: {
         stopButton.state = 'stopOn'
-        if (playPause.isConnected()) {
-            playPause.stopClicked()
-        }
     }
 
     // Specify the behavior of a button when it is clicked on
     onReleased: {
         stopButton.state = ''
+        if (playPause.isConnected()) {
```

Comment:
Isn't there  a onClicked action that matches that more closely ? but maybe it doesn't do what you want it to do